### ISSUE TITLE
Need to integration with translation system

### ISSUE THREAD

**Issue by robert-ancell** [canonical/lightdm#258]
In the past translations were done in [Launchpad](https://launchpad.net/), however since migrating to GitHub this no longer works. We need to either get that working again or use some other translation service to provide translations.

Doing translations via PRs is not useful since maintainers have no way of knowing if the translations are correct / better.

**Comment by fitojb** [comment]
Weblate!

**Comment by robert-ancell** [comment]
Alternatively if no-one is able to produce a PR to resolve this I would be happy to give someone permissions to land translations if they are able to validate them (i.e. they know the translation community and can tell if contributors have a good reputation for changes).

**Comment by hosiet** [comment]
I am a bit concerned with using translation service vs. enforcing signing Canonical CLA. In contrast to GitHub PRs, the translation service will be very hard to confirm that the contributor has signed CLA. You will either need to get things customized or make an exception on CLA signing.

**Comment by fitojb** [comment]
Weblate can enforce that aspect @hosiet

**Comment by NorwayFun** [comment]
Hello,
Any news regarding this topic, pretty please?
I'm tryng to get Georgian translation in for a year now :)

Temuri

**Comment by hosiet** [comment]
I see no reason preventing people from updating translations via PR. Even if the project uses some translation service, will the maintainer magically know that the translation is right or wrong? I highly doubt that, no matter using Launchpad, Weblate or any other hosted translation websites.

**Comment by NorwayFun** [comment]
I tried to use launchpad, but for this particular case it hassome cryptic queston like "my manager name" etc.
In case there is way to get translations in from launchpad, please, let me know.

**Comment by NorwayFun** [comment]
Hello,
Any news about this topic, please?

**Comment by hosiet** [comment]
Another 6 months have passed, and this issue hasn't been solved since 2022. I would like to emphasize again:

I see no reason preventing people from updating translations via PR. Asking for CLA signing is fine.

Even if the project uses some translation service, will the maintainer magically know that the translation is right or wrong? I highly doubt that, no matter using Launchpad, Weblate or any other hosted translation websites.

**Comment by NorwayFun** [comment]
Another 4 months. Any news?

**Comment by hosiet** [comment]
Yet another 6 months have passed, and this issue hasn't been solved since 2022. I would like to emphasize again @robert-ancell :

I see no reason preventing people from updating translations via PR. Asking for CLA signing is fine.

Even if the project uses some translation service, will the maintainer magically know that the translation is right or wrong? I highly doubt that, no matter using Launchpad, Weblate or any other hosted translation websites.

If you are *really* that concerned, I would suggest starting to grant individuals with exceptions on handling of each language's translation (instead of a person that covers all languages). For example, I myself as seen from https://launchpad.net/~hosiet/+participation is a member of Launchpad CLA-signed users, Ubuntu China member, Ubuntu Simplified Chinese translator and Launchpad Simplified Chinese translator. If individual has both CLA signed and some role in existing translation memberships (especially via Launchpad), their contribution of translation should be accepted via PR. In the meanwhile, anyone can raise translation concerns via issues in case the individual made any mistake.

TL;DR: Doing nothing is the worst choice.
